librz/core/cconfig.c

```diff
@@ -89,25 +89,21 @@ static bool cb_vmlinux(void *_core, void *_node) {
 	core->analysis->vmlinux_config = rz_vmlinux_config_new();
 	core->analysis->vmlinux_config->config_tbl = rz_vmlinux_config_table_new();
 
-	const char* vmlinux_config = rz_config_get(core->config, "bin.elf.vmlinux.config");
-	const char* vmlinux_version = rz_config_get(core->config, "bin.elf.vmlinux.version");
+	const char *vmlinux_config = rz_config_get(core->config, "bin.elf.vmlinux.config");
+	const char *vmlinux_version = rz_config_get(core->config, "bin.elf.vmlinux.version");
 
-	printf("Parsing config file '%s'\n", vmlinux_config);
+	RZ_LOG_INFO("Parsing config file '%s'...\n", vmlinux_config);
 	vmlinux_parse_apply_config_file(vmlinux_config, core->analysis->vmlinux_config->config_tbl);
```

Comment:
maybe before parsing, you should check if the string is empty and if the file exists.

Reply:
Hello. I am verifying it inside the function `vmlinux_parse_apply_config_file`